pandas/core/groupby/ops.py

```diff
@@ -607,9 +609,10 @@ def _call_cython_op(
                     min_count=min_count,
                     mask=mask,
                     result_mask=result_mask,
+                    **kwargs,
                 )
             else:
-                func(result, counts, values, comp_ids, min_count, **kwargs)
+                func(result, counts, values, comp_ids, min_count)
```

Comment:
kwargs is always empty here, is that right? I would recommend still passing it through or adding `assert not kwargs` to ensure we don't accidentally not pass through if things change upstream (would be a tough bug to find)

Reply:
I added this recently when removing the special case for ddof != 1 and I think the only function that is still called here does not support more arguments, so that would be more bad than good?

Reply:
gentle ping @rhshadrach

Reply:
Thanks for the ping - I'm not sure I understand. When we get to this line, kwargs should be empty and if they are not, something has gone wrong (I think). If this were to happen, the code would run without error (since kwargs are no longer passed to `func`) and thus be silently "failing". When I see such a possibility, I like to add asserts so that if we get to an erroneous state, the failure is no longer silent.

But like I said, I'm also good as-is.

Reply:
Ah got you. Yes this is true. Would keep it like this till we move the remaining functions up. Could also add Kwargs but would have to do it for every function call then I think

Reply:
Sounds good

Reply:
Which one? :)